extensions/objects/templates/FilepathEditor.js

```diff
@@ -36,6 +36,11 @@ oppia.directive('filepathEditor', [
         $scope.MODE_SAVED = 3;
         $scope.MAX_OUTPUT_IMAGE_WIDTH_PX = 490;
 
+        // We only use PNG format since that is what canvas can export to in
+        // all browsers.
+        // TODO(sll): See if we can add support for other image formats.
+        var RESAMPLED_IMAGE_FORMAT = 'png';
```

Comment:
I use $scope for stuff that's shared with the directive template, and var for stuff that isn't. It looks like your vars are private to the controller, so I can change yours to var too to maintain consistency? Sorry for not catching this last time.